### DISCUSSION TITLE
How to Reset Auto-Login Memory

### DISCUSSION THREAD

**Discussion by zacaronii** [cortexrd/Knack-Toolkit-Library#395] [Q&A] [answered]
I setup _al on a page for a kiosk and then I accidentally didn't accept the popup initially. I am using auto-login for a kiosk mode setup on an iPad, and I can't figure out how to get the auto-login to prompt again. I tried clearing history, cookies, etc.

Apologies if I missed this somewhere in the docs or in a previous discussion, but I feel like I've searched thoroughly at this point. I'm sure it's easy enough, and I just lack the understanding on how to refresh it.

**Comment by cortexrd** [comment, marked as answer]
Easy!  Two steps:

1. Click on the Version Info Bar
2. Click on Reset Auto-Login

![image](https://github.com/user-attachments/assets/87212dad-be09-48fa-9361-8d5377f38635)

Norm

**Reply by cortexrd** [reply]
@zacaronii
Can you mark **this post above** as the answer please?
Thanks :)

**Reply by zacaronii** [reply]
Yes, sorry I meant to come back and do that, but it completely slipped my mind. Thanks, again!

**Comment by cortexrd** [comment]
BTW, there's an intermittent bug when setting Auto-Login.  Sometimes it works the first time, but sometimes you get in a loop and have to do it 2-3 times.  I didn't have time to figure out what causes this, but will fix it eventually.  Let me know if you see this behavior.

ND

**Comment by zacaronii** [comment]
Well, I'm feeling rather foolish for not figuring that out for myself. Thank you so much for the prompt response, Norm!
